src/app/component/auth/auth.module.ts

```diff
@@ -1,19 +1,22 @@
 import { NgModule } from '@angular/core';
```

Comment:
general: Please name branches according to the agreed terms of the name(features - feature/*, bugs - bug/*)

Reply:
ok